diff -rup libnl-tiny-svn20042015-orig/files/libnl-tiny.pc libnl-tiny-svn20042015/files/libnl-tiny.pc
--- libnl-tiny-svn20042015-orig/files/libnl-tiny.pc	2015-04-20 13:54:01.747892999 -0400
+++ libnl-tiny-svn20042015/files/libnl-tiny.pc	2015-04-20 13:54:06.399892999 -0400
@@ -1,5 +1,5 @@
-prefix=/usr
-exec_prefix=/usr
+prefix=
+exec_prefix=
 libdir=${exec_prefix}/lib
 includedir=${prefix}/include/libnl-tiny
 
diff -rup libnl-tiny-svn20042015-orig/src/Makefile libnl-tiny-svn20042015/src/Makefile
--- libnl-tiny-svn20042015-orig/src/Makefile	2015-04-20 13:54:01.747892999 -0400
+++ libnl-tiny-svn20042015/src/Makefile	2015-04-20 13:55:15.679892999 -0400
@@ -1,9 +1,10 @@
-CC=gcc
+CC?=gcc
+AR?=ar
 WFLAGS=-Wall
-CFLAGS=-O2
+CFLAGS?=-O2
 INCLUDES=-Iinclude
 
-LIBNAME=libnl-tiny.so
+LIBNAME=libnl-tiny.a
 
 all: $(LIBNAME)
 
@@ -14,4 +15,4 @@ LIBNL_OBJ=nl.o handlers.o msg.o attr.o c
 GENL_OBJ=genl.o genl_family.o genl_ctrl.o genl_mngt.o unl.o
 
 $(LIBNAME): $(LIBNL_OBJ) $(GENL_OBJ)
-	$(CC) -Wl,-Bsymbolic-functions -shared -o $@ $^
+	$(AR) rcs $@ $^
