diff -rup mutt-1.5.24-orig/Makefile.am mutt-1.5.24/Makefile.am
--- mutt-1.5.24-orig/Makefile.am	2015-09-14 12:59:07.105781413 +0300
+++ mutt-1.5.24/Makefile.am	2015-09-14 12:59:38.085934778 +0300
@@ -10,7 +10,7 @@ IMAP_SUBDIR = imap
 IMAP_INCLUDES = -I$(top_srcdir)/imap
 endif
 
-SUBDIRS = m4 po intl doc contrib $(IMAP_SUBDIR)
+SUBDIRS = m4 po intl doc $(IMAP_SUBDIR)
 
 bin_SCRIPTS = muttbug flea $(SMIMEAUX_TARGET)
 
