diff -rup stag-git31012015-orig/Makefile stag-git31012015/Makefile
--- stag-git31012015-orig/Makefile	2015-01-31 00:03:11.569131214 +0200
+++ stag-git31012015/Makefile	2015-01-31 00:03:24.805131529 +0200
@@ -1,25 +1,24 @@
 CC ?= gcc
-CFLAGS=-Wall -Werror -Wextra -std=c99 -pedantic -Wno-unused-parameter
-# D_BSD_SOURCE for strsep
+CFLAGS+=-Wall -Werror -Wextra -std=c99 -pedantic -Wno-unused-parameter -D_BSD_SOURCE
 LIBS=-lncurses -lm -D_DEFAULT_SOURCE
-PREFIX ?= /usr/local
+PREFIX ?=
 
 .PHONY: install uninstall clean
 
 all: stag
 
 stag:
-	$(CC) $(CFLAGS) stag.c view.c data.c -o stag $(LIBS) 
+	$(CC) $(CFLAGS) stag.c view.c data.c -o stag $(LDFLAGS) $(LIBS) 
 
 install:
-	install -d "$(DESTDIR)$(PREFIX)/bin"
-	install -d "$(DESTDIR)$(PREFIX)/share/man/man1"
-	install -m755 stag "$(DESTDIR)$(PREFIX)/bin/stag"
-	install -m644 doc/stag.1 "$(DESTDIR)$(PREFIX)/share/man/man1/stag.1"
+	install -d "$(DESTDIR)/bin"
+	install -d "$(DESTDIR)$(PREFIX)/man/man1"
+	install -m755 stag "$(DESTDIR)/bin/stag"
+	install -m644 doc/stag.1 "$(DESTDIR)$(PREFIX)/man/man1/stag.1"
 
 uninstall:
 	rm "$(DESTDIR)$(PREFIX)/bin/stag"
-	rm "$(DESTDIR)$(PREFIX)/share/man/man1/stag.1"
+	rm "$(DESTDIR)$(PREFIX)/man/man1/stag.1"
 
 clean:
 	rm -f *.o stag
