diff -rup libjpeg-turbo-1.4.1-orig/Makefile.in libjpeg-turbo-1.4.1/Makefile.in
--- libjpeg-turbo-1.4.1-orig/Makefile.in	2015-06-09 20:59:22.221803249 +0300
+++ libjpeg-turbo-1.4.1/Makefile.in	2015-06-09 21:02:58.450875410 +0300
@@ -414,7 +414,7 @@ build_cpu = @build_cpu@
 build_os = @build_os@
 build_vendor = @build_vendor@
 datadir = @datadir@
-docdir = @docdir@
+docdir = @docdir@/@PACKAGE@
 exec_prefix = @exec_prefix@
 host = @host@
 host_alias = @host_alias@
