diff -rup musl-1.1.11-orig/src/ldso/dynlink.c musl-1.1.11/src/ldso/dynlink.c
--- musl-1.1.11-orig/src/ldso/dynlink.c	2015-09-14 12:47:59.590471389 +0300
+++ musl-1.1.11/src/ldso/dynlink.c	2015-09-14 12:50:20.591170636 +0300
@@ -1251,7 +1251,6 @@ _Noreturn void __dls3(size_t *sp)
 	/* Only trust user/env if kernel says we're not suid/sgid */
 	if (!libc.secure) {
 		env_path = getenv("LD_LIBRARY_PATH");
-		env_preload = getenv("LD_PRELOAD");
 	}
 
 	/* If the main program was already loaded by the kernel,
