diff -rup vitetris-0.57-orig/Makefile vitetris-0.57/Makefile
--- vitetris-0.57-orig/Makefile	2015-02-27 18:46:17.767183037 +0200
+++ vitetris-0.57/Makefile	2015-02-27 18:46:41.499183601 +0200
@@ -1,9 +1,9 @@
 include config.mk
 
-PROGNAME = tetris$(EXE)
+PROGNAME = vitetris$(EXE)
 
 # Uncomment to change the default.  (Only used in Unix-like systems.)
-#HISCORE_FILENAME = /var/games/vitetris-hiscores
+HISCORE_FILENAME = /var/games/vitetris/hiscores
 
 INSTALL = install -oroot -groot
 
@@ -17,8 +17,6 @@ $(PROGNAME):
 build: src/src-conf.mk
 	cd src; $(MAKE) tetris
 	mv -f src/tetris$(EXE) $(PROGNAME)
-	@echo stripping symbols to reduce program size:
-	-strip --strip-all $(PROGNAME)
 
 gameserver: src/netw/gameserver.c
 	cd src/netw; $(MAKE) gameserver
