diff -rup snownews-1.5.12-orig/configure snownews-1.5.12/configure
--- snownews-1.5.12-orig/configure	2015-01-30 15:20:41.394232819 +0200
+++ snownews-1.5.12/configure	2015-01-30 15:20:47.358232961 +0200
@@ -12,7 +12,7 @@ my $xmlldflags = `xml2-config --libs`;
 chomp($xmlldflags);
 
 my $prefix = "/usr/local";
-my $cflags = "-Wall -Wno-format-y2k -O2 -DLOCALEPATH=\"\\\"\$(LOCALEPATH)\\\"\" -DOS=\\\"$os\\\" $xmlcflags \$(EXTRA_CFLAGS) ";
+my $cflags = "-Wall -Wno-format-y2k -DLOCALEPATH=\"\\\"\$(LOCALEPATH)\\\"\" -DOS=\\\"$os\\\" $xmlcflags \$(EXTRA_CFLAGS) ";
 my $ldflags = "-lncurses -lcrypto $xmlldflags \$(EXTRA_LDFLAGS) ";
 
 my $use_nls = 1;
diff -rup snownews-1.5.12-orig/Makefile snownews-1.5.12/Makefile
--- snownews-1.5.12-orig/Makefile	2015-01-30 15:20:41.398232819 +0200
+++ snownews-1.5.12/Makefile	2015-01-30 15:21:07.874233449 +0200
@@ -1,8 +1,8 @@
-CC=         gcc
+CC?=         gcc
 MAKE=       make
 INSTALL=    install
-LOCALEPATH= $(PREFIX)/share/locale
-MANPATH=    $(PREFIX)/share/man
+LOCALEPATH= $(PREFIX)/locale
+MANPATH=    $(PREFIX)/man
 
 ### Compiler/linker flags   ###
 ### Generated via configure ###
@@ -20,11 +20,11 @@ DISTFILES = AUTHOR COPYING CREDITS Chang
 	config.h version.h main.h interface.h netio.h xmlparse.h updatecheck.h os-support.h conversions.h dialog.h ui-support.h categories.h about.h cookies.h setup.h net-support.h digcalc.h filters.h io-internal.h zlib_interface.h support.h
 
 ### Translations ###
-LOCALES= de es fr it nl ru sl se zh_TW zh_CN pt_BR pl ja be@latin uk_UA
+LOCALES=
 LOC=     po
 
 ### Manpages ##
-LANGS= de fr it nl ru_RU.KOI8-R
+LANGS=
 MAN=   doc/man
 
 ### Compile ###
