diff -rup atop-2.2-3-orig/Makefile atop-2.2-3/Makefile
--- atop-2.2-3-orig/Makefile	2015-06-30 18:19:18.181320319 +0300
+++ atop-2.2-3/Makefile	2015-06-30 18:22:06.366154302 +0300
@@ -132,10 +132,6 @@ genericinstall:	atop atopacctd
 		then	mkdir -p $(DESTDIR)$(MAN5PATH);	fi
 		if [ ! -d $(DESTDIR)$(MAN8PATH) ]; 		\
 		then	mkdir -p $(DESTDIR)$(MAN8PATH);	fi
-		if [ ! -d $(DESTDIR)$(CRNPATH) ]; 		\
-		then	mkdir -p $(DESTDIR)$(CRNPATH);	fi
-		if [ ! -d $(DESTDIR)$(ROTPATH) ]; 		\
-		then	mkdir -p $(DESTDIR)$(ROTPATH);	fi
 		#
 		cp atop   		$(DESTDIR)$(BINPATH)/atop
 		chown root		$(DESTDIR)$(BINPATH)/atop
@@ -144,18 +140,10 @@ genericinstall:	atop atopacctd
 		cp atopacctd  		$(DESTDIR)$(SBINPATH)/atopacctd
 		chown root		$(DESTDIR)$(SBINPATH)/atopacctd
 		chmod 0700 		$(DESTDIR)$(SBINPATH)/atopacctd
-		cp atop   		$(DESTDIR)$(BINPATH)/atop-$(VERS)
-		ln -sf atop-$(VERS)     $(DESTDIR)$(BINPATH)/atopsar-$(VERS)
-		cp atop.daily    	$(DESTDIR)$(SCRPATH)
-		chmod 0711 	 	$(DESTDIR)$(SCRPATH)/atop.daily
 		cp man/atop.1    	$(DESTDIR)$(MAN1PATH)
 		cp man/atopsar.1 	$(DESTDIR)$(MAN1PATH)
 		cp man/atoprc.5  	$(DESTDIR)$(MAN5PATH)
 		cp man/atopacctd.8  	$(DESTDIR)$(MAN8PATH)
-		cp psaccs_atop   	$(DESTDIR)$(ROTPATH)/psaccs_atop
-		cp psaccu_atop  	$(DESTDIR)$(ROTPATH)/psaccu_atop
-		touch          	  	$(DESTDIR)$(LOGPATH)/dummy_before
-		touch            	$(DESTDIR)$(LOGPATH)/dummy_after
 
 ##########################################################################
 
