diff -rup dvtm-0.14-orig/config.mk dvtm-0.14/config.mk
--- dvtm-0.14-orig/config.mk	2015-02-28 16:35:20.850125031 +0200
+++ dvtm-0.14/config.mk	2015-02-28 16:37:07.306127561 +0200
@@ -3,8 +3,8 @@ VERSION = 0.14
 
 # Customize below to fit your system
 
-PREFIX ?= /usr/local
-MANPREFIX = ${PREFIX}/share/man
+PREFIX =
+MANPREFIX = ${PREFIX}/man
 # specify your systems terminfo directory
 # leave empty to install into your home folder
 TERMINFO := ${DESTDIR}${PREFIX}/share/terminfo
