diff -rup linux-3.14.37-orig/drivers/tty/vt/vt.c linux-3.14.37/drivers/tty/vt/vt.c
--- linux-3.14.37-orig/drivers/tty/vt/vt.c	2015-04-06 11:57:23.347161524 +0300
+++ linux-3.14.37/drivers/tty/vt/vt.c	2015-04-06 11:58:05.951162537 +0300
@@ -2822,8 +2822,8 @@ static void con_shutdown(struct tty_stru
 	console_unlock();
 }
 
-static int default_color           = 7; /* white */
-static int default_italic_color    = 2; // green (ASCII)
+static int default_color           = 2; /* green */
+static int default_italic_color    = 7; // white (ASCII)
 static int default_underline_color = 3; // cyan (ASCII)
 module_param_named(color, default_color, int, S_IRUGO | S_IWUSR);
 module_param_named(italic, default_italic_color, int, S_IRUGO | S_IWUSR);
