diff -rup linux-4.1-orig/scripts/Makefile.lib linux-4.1/scripts/Makefile.lib
--- linux-4.1-orig/scripts/Makefile.lib	2015-06-22 11:44:45.145171520 +0300
+++ linux-4.1/scripts/Makefile.lib	2015-06-22 11:47:54.170108845 +0300
@@ -386,5 +386,5 @@ cmd_xzkern = (cat $(filter-out FORCE,$^)
 
 quiet_cmd_xzmisc = XZMISC  $@
 cmd_xzmisc = (cat $(filter-out FORCE,$^) | \
-	xz --check=crc32 --lzma2=dict=1MiB) > $@ || \
+	xz -9 -e --check=crc32 --lzma2=dict=1MiB) > $@ || \
 	(rm -f $@ ; false)
diff -rup linux-4.1-orig/scripts/xz_wrap.sh linux-4.1/scripts/xz_wrap.sh
--- linux-4.1-orig/scripts/xz_wrap.sh	2015-06-22 11:44:45.129171436 +0300
+++ linux-4.1/scripts/xz_wrap.sh	2015-06-22 11:47:54.170108845 +0300
@@ -20,4 +20,4 @@ case $SRCARCH in
 	sparc)          BCJ=--sparc ;;
 esac
 
-exec xz --check=crc32 $BCJ --lzma2=$LZMA2OPTS,dict=32MiB
+exec xz -9 -e --check=crc32 $BCJ --lzma2=$LZMA2OPTS,dict=32MiB
