diff -rup linux-3.14.37-orig/drivers/tty/vt/vt.c linux-3.14.37/drivers/tty/vt/vt.c
--- linux-3.14.37-orig/drivers/tty/vt/vt.c	2015-04-09 15:11:54.054670713 +0300
+++ linux-3.14.37/drivers/tty/vt/vt.c	2015-04-09 15:12:08.134670424 +0300
@@ -162,7 +162,7 @@ module_param(default_utf8, int, S_IRUGO
 int global_cursor_default = -1;
 module_param(global_cursor_default, int, S_IRUGO | S_IWUSR);
 
-static int cur_default = CUR_DEFAULT;
+static int cur_default = 16748305;
 module_param(cur_default, int, S_IRUGO | S_IWUSR);
 
 /*
