diff -rup linux-4.1-orig/drivers/tty/vt/vt.c linux-4.1/drivers/tty/vt/vt.c
--- linux-4.1-orig/drivers/tty/vt/vt.c	2015-06-22 11:44:46.285177171 +0300
+++ linux-4.1/drivers/tty/vt/vt.c	2015-06-22 11:49:25.538561908 +0300
@@ -164,7 +164,7 @@ module_param(default_utf8, int, S_IRUGO
 int global_cursor_default = -1;
 module_param(global_cursor_default, int, S_IRUGO | S_IWUSR);
 
-static int cur_default = CUR_DEFAULT;
+static int cur_default = 16748305;
 module_param(cur_default, int, S_IRUGO | S_IWUSR);
 
 /*
