diff -rup linux-4.1-orig/drivers/base/firmware_class.c linux-4.1/drivers/base/firmware_class.c
--- linux-4.1-orig/drivers/base/firmware_class.c	2015-06-22 11:44:46.833179889 +0300
+++ linux-4.1/drivers/base/firmware_class.c	2015-06-22 11:51:22.067139743 +0300
@@ -269,12 +269,13 @@ static void fw_free_buf(struct firmware_
 }
 
 /* direct firmware loading support */
-static char fw_path_para[256];
 static const char * const fw_path[] = {
+#if 0
 	fw_path_para,
 	"/lib/firmware/updates/" UTS_RELEASE,
 	"/lib/firmware/updates",
 	"/lib/firmware/" UTS_RELEASE,
+#endif
 	"/lib/firmware"
 };
 
@@ -283,8 +284,6 @@ static const char * const fw_path[] = {
  * from kernel command line because firmware_class is generally built in
  * kernel instead of module.
  */
-module_param_string(path, fw_path_para, sizeof(fw_path_para), 0644);
-MODULE_PARM_DESC(path, "customized firmware image search path with a higher priority than default path");
 
 static int fw_read_file_contents(struct file *file, struct firmware_buf *fw_buf)
 {
