diff -rup linux-4.1-orig/mm/vmscan.c linux-4.1/mm/vmscan.c
--- linux-4.1-orig/mm/vmscan.c	2015-06-22 18:45:41.850486245 +0300
+++ linux-4.1/mm/vmscan.c	2015-06-22 18:46:48.930818877 +0300
@@ -139,7 +139,7 @@ struct scan_control {
 /*
  * From 0 .. 100.  Higher means more swappy.
  */
-int vm_swappiness = 60;
+int vm_swappiness = 20;
 /*
  * The total number of pages which are beyond the high watermark within all
  * zones.
