diff -rup linux-4.1-orig/include/linux/printk.h linux-4.1/include/linux/printk.h
--- linux-4.1-orig/include/linux/printk.h	2015-06-22 11:44:44.633168976 +0300
+++ linux-4.1/include/linux/printk.h	2015-06-22 12:07:45.524016450 +0300
@@ -37,7 +37,7 @@ static inline const char *printk_skip_le
 #define CONSOLE_LOGLEVEL_SILENT  0 /* Mum's the word */
 #define CONSOLE_LOGLEVEL_MIN	 1 /* Minimum loglevel we let people use */
 #define CONSOLE_LOGLEVEL_QUIET	 4 /* Shhh ..., when booted with "quiet" */
-#define CONSOLE_LOGLEVEL_DEFAULT 7 /* anything MORE serious than KERN_DEBUG */
+#define CONSOLE_LOGLEVEL_DEFAULT 3 /* anything MORE serious than KERN_ERR */
 #define CONSOLE_LOGLEVEL_DEBUG	10 /* issue debug messages */
 #define CONSOLE_LOGLEVEL_MOTORMOUTH 15	/* You can't shut this one up */
 
