diff -rup vifm-0.8-orig/src/Makefile.in vifm-0.8/src/Makefile.in
--- vifm-0.8-orig/src/Makefile.in	2015-07-11 11:05:04.557977851 +0300
+++ vifm-0.8/src/Makefile.in	2015-07-11 11:08:13.094912778 +0300
@@ -348,7 +348,7 @@ MOSTLYCLEANFILES = compile_info.c \
 
 desktopappdir = $(datadir)/applications
 desktopappfile = ../data/vifm.desktop
-desktopapp_DATA = $(desktopappfile)
+desktopapp_DATA =
 desktoppixdir = $(datadir)/pixmaps
 desktoppixfile = ../data/graphics/vifm.png
 desktoppix_DATA = $(desktoppixfile)
