diff -rup libressl-2.2.0-orig/crypto/compat/getentropy_linux.c libressl-2.2.0/crypto/compat/getentropy_linux.c
--- libressl-2.2.0-orig/crypto/compat/getentropy_linux.c	2015-06-12 10:18:16.053025369 +0300
+++ libressl-2.2.0/crypto/compat/getentropy_linux.c	2015-06-12 10:36:25.990430097 +0300
@@ -28,7 +28,6 @@
 #include <sys/resource.h>
 #include <sys/syscall.h>
 #ifdef SYS__sysctl
-#include <sys/sysctl.h>
 #include <linux/sysctl.h>
 #endif
 #include <sys/statvfs.h>
